### PR TITLE
PR for Add Cart Page and Wishlist Page

### PR DESCRIPTION
**Pull request by manojasarsa**
This PR includes two pages :

1. Cart Management Page - Products present in cart.

2. Wishlist Management Page - Products present in wishlist.

### CODE REVIEW

File: header.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Round");
+@import url('https://fonts.googleapis.com/css2?family=Marck+Script&family=Tangerine&display=swap');
+
+.main_header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    border-bottom: 0;
+    z-index: 999;
```
Comment (srishti-maurya) [inline header.css]: Maybe you may give a bit lower `z-index` value

**Review by srishti-maurya — APPROVED** [review]
UI looks clean and good.👍
I hope the below badge text and image not overlapping is intentional. If not, you can refactor it maybe.

Screenshot 2022-02-16 at 11 24 10 PM

File: cart-and-wishlist/cart.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+.cart_title {
+    text-align: center;
+    margin: 2rem 0;
+}
+
+.cart_qty {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (vaibhav18matere) [inline cart-and-wishlist/cart.css:70]: you can use a separate class for this, same as you used in component library

File: cart-and-wishlist/cart.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    <link rel="stylesheet" href="https://zero-ui.netlify.app/docs/docs.css">
+    <title>Cart Page</title>
+</head>
+
+<body>
+
+    <div class="cart_wrapper">
+
+        <!-- Header -->
+
```
Comment (vaibhav18matere) [inline cart-and-wishlist/cart.html]: you can remove these blank lines

File: cart-and-wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="/cart-and-wishlist/wishlist.css">
+    <link rel="stylesheet" href="/cart-and-wishlist/cart.css">
+    <link rel="stylesheet" href="/header.css">
+
+    <script src="https://kit.fontawesome.com/b2c3fdb829.js" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://zero-ui.netlify.app/docs/docs.css">
+
```
Comment (vaibhav18matere) [inline cart-and-wishlist/wishlist.html]: add css file before js file

File: header.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    padding: 0.5rem 0.6rem;
+    border-radius: 0;
+    border: none;
+    border: 1px solid var(--shadow-color);
+}
+
+.header .nav_right {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (vaibhav18matere) [inline header.css:42]: here you can use pre-built css class

File: header.css (hunk 1 above)
Comment (manojasarsa) [reply header.css]: Yea, I'll do that. 

> UI looks clean and good.👍 I hope the below badge text and image not overlapping is intentional. If not, you can refactor it maybe.
> 
> Screenshot 2022-02-16 at 11 24 10 PM



> UI looks clean and good.👍 I hope the below badge text and image not overlapping is intentional. If not, you can refactor it maybe.
> 
> Screenshot 2022-02-16 at 11 24 10 PM

Oh I think I need to refactor it. Thanks, Srishti for pointing out.

**Review by vaibhav18matere — APPROVED** [review]
everything is clean and impressive, here's small suggestions

File: cart-and-wishlist/cart.css (hunk 2 above)
Comment (manojasarsa) [reply cart-and-wishlist/cart.css:70]: Okay, Vaibhav. Will do that :)

**Comment by preethasri** [comment]
UI looks good .and the brand name font looks unique👍

**Pull request merged**